packages/block-editor/src/components/inserter/style.scss

```diff
@@ -143,11 +143,13 @@ $block-inserter-search-height: 38px;
 	margin-right: 20px;
 	padding: 20px;
 	background: $white;
+	box-shadow: $shadow-popover;
+	border-radius: $radius-block-ui;
 
 	@include break-medium {
 		position: absolute;
-		top: 0;
-		left: calc(100% + 20px);
+		top: -1px;
+		left: calc(100% + 12px);
```

Comment:
You can use `$grid-unit-15`here instead of 12px, as it's part of the grid spacing!

Because it's part of a calc rule, you may have to do this: `left: calc(100% + #{$grid-unit-15});`